### DISCUSSION TITLE
How to debug code in Visual Studio?

### DISCUSSION THREAD

**Discussion by ZhengZQ007** [OpenRadioss/OpenRadioss#1769] [Q&A]
Hey, I have just build OpenRadioss with Visual Studio in debug mode and set some breakpoints. The engine program (engine_win64_db.exe) can not stop at the breakpoints when I press F5 (run with debug), while the starter program (starter_win64_db.exe) can stop at the breakpoints. What is the correct way to debug OpenRadioss in Visual Studio?

**Comment by OlivierWienholtz** [comment]
Hello,

I am able to perform Engine Debugs. 
Debugging in Engine is same than Starter.
Pls ensure the symbol files (.pdb) which are in [repo Root]\vs_build_win64\Engine_x64-Debug\engine\ 
are set correctly & load, Like I believe you did with Starter.

**Reply by ZhengZQ007** [reply]
Hi Olivier,

Thanks for your answer. I rebuild the Engine Debugs and can debug correctly. But I meet another trouble when I want to watch the local variables in debug mode. My visual studio just keeps busy in evaluating expressions for a long time. Have you met the same trouble?

Best Regards.

Zheng.

**Reply by ZhengZQ007** [reply]
I re-installed my building environment and now can debug the code as usual.
